public/js/visualisation_helpers.js

```diff
@@ -0,0 +1,57 @@
+import _ from 'underscore';
+
+export default class VisualisationHelpers {
```

Comment:
We don't want a class here. Just a collection of methods in a file. Import in graph.js or length_visualisation.js as `import * as Helpers from './visualisation_helpers.js'` and use as `Helpers.tick_formatter`.